wfe/wfe.go

```diff
@@ -2079,6 +2089,17 @@ func (wfe *WebFrontEndImpl) Authz(
 				response)
 			return
 		}
+
+		if authz.Status == acme.StatusPending {
+			// Check for the existence of a challenge which state is processing, and add the
+			// Retry-After header if there is one.
+			for _, c := range authz.Challenges {
+				if c.Status == acme.StatusProcessing {
```

Comment:
As mentioned in [this comment](https://github.com/letsencrypt/pebble/pull/380#discussion_r865422209), the `processing` state for challenges is not currently implemented in Pebble. 

Consequently, the `Retry-After` header is never sent for challenges.

I think we'd need to set the state to `processing` right as the challenge is queued up for validation in `wfe.updateChallenge`. Do we want to do this in a separate PR or here?

Reply:
@alexzorin 
There was an error in my understanding. My apologies.
I now understand that `processing` status is not currently used for `challenge` in pebble.
Regarding the unused `processing`, I felt that perhaps it would be appropriate to proceed with implementation in another PR.